src/blender_executable.ts

```diff
@@ -180,6 +180,10 @@ async function getFilteredBlenderPath(type: BlenderType): Promise<BlenderPathDat
     // update VScode settings
     if (settingsBlenderPaths.find(data => data.path === pathData.path) === undefined) {
         settingsBlenderPaths.push(pathData);
+        // do not save linuxInode is settings
+        for (const settingExe of settingsBlenderPaths) {
+            settingExe.linuxInode = undefined;
```

Comment:
Maybe it's better to explicitly mention the properties that should be stored instead of excluding some? That makes it more obvious what's going on imo.